src/bundle/Resources/public/js/scripts/sidebar/btn/user.edit.js

```diff
@@ -0,0 +1,16 @@
+(function (global, doc) {
+    const editButton = doc.querySelector('.ez-btn--edit-user');
+    const languageRadioOption = doc.querySelector('.ez-extra-actions--edit-user.ez-extra-actions--prevent-show [type="radio"]');
+    const editActions = doc.querySelector('.ez-extra-actions--edit-user');
+    const form = editActions.querySelector('form');
+
+    if (!languageRadioOption) {
```

Comment:
Why in edit form there is no language at all?

Reply:
connected to `ez-extra-actions--prevent-show` class.